### DISCUSSION TITLE
Trains design window: Do not unselect after copy

### DISCUSSION THREAD

**Discussion by Nihildum** [voxeltycoon/issues#331] [General]
If you create a train and add wagon, then you need to copy it several times. For this process you need to:
 - Select the wagon
 - Move your mouse to the copy button
 - Click copy
 - Move your mouse to the wagon again
 - Repeat like 4 to 20 times.

An improved process would be to not unselect the wagon after copy is clicked. This could also be implemented in such a way, that when the user pressed shift while clicking on copy, that selection is kept. (This could be added as an tooltip.)

**Comment by katepolak** [comment]
You can already do this easily by selecting multiple carriges to copy, but shift+click would also make sense to implement as that's the behaviour of other copy-like operations in the game.

**Reply by Nihildum** [reply]
I just found out, that you can add multiple wagons by shift clicking in the window which adds new wagons.

Making this more obvious would help.

You see it on the tooltip after you press shift, showing this on the tooltip bevorhand would help.
